### PR TITLE
bootstrap: cleanup status port and config dir

### PR DESCRIPTION
**Pull request by kyessenov**
Signed-off-by: Kuat Yessenov <kuat@google.com>

- change "15021" to a UDS port
- configure gateways to use proxy config status port 15021 for compatibility
- use ConfigPath instead of hardcoded paths in bootstrap template

Relevant link https://docs.google.com/document/d/1xHy2jQ8oiwMponMVY2zJr2eUAmHW_Hi9JK42a7cg5Pc/edit#bookmark=id.uzcgjyodob15

### CODE REVIEW

**Review by howardjohn — CHANGES_REQUESTED** [review]
This will break external load balancer health checks, which rely on hitting this from outside of the pod

**Comment by kyessenov** [comment]
@howardjohn Let's discuss this offline. I'll ping you later.

File: manifests/profiles/demo.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ spec:
             # on this list. Setting this to the health check port will ensure that health
             # checks always work. https://github.com/istio/istio/issues/12503
             - port: 15021
-              targetPort: 15021
+              targetPort: 15020
```
[the remark this answers is not in the document]
Comment (kyessenov) [reply manifests/profiles/demo.yaml:31]: Main docs for GLBC use different target port https://cloud.google.com/kubernetes-engine/docs/how-to/load-balance-ingress. Are you sure about lack of support?

**Comment by kyessenov** [comment]
@howardjohn Any idea what is this hybrid using 1.9 injection template with 1.10 proxy container? Why does a new proxy pick up old control plane?

**Comment by kyessenov** [comment]
I think I'll abandon this approach. It's truly a mess with two status ports, so we're stuck with them.

**Pull request closed without merging**